src/lucky/cookies/session_cookie.cr

```diff
@@ -0,0 +1,51 @@
+class Lucky::SessionCookie
+  Habitat.create do
+    setting key : String
+  end
+
+  alias Key = String | Symbol
+  private property store = {} of String => String
+  @changed = false
+
+  def initialize(cookie : Lucky::MaybeCookie = Lucky::NullCookie.new)
```

Comment:
I’m a bit nervous about initializing this with a NullCookie. It seems like that might be set on accident. What do you think about requiring setting the cookie and having another class method that sets null cookie:

def self.null
  new.tap do |session|
    session.store = NullCookie.new
  end
end

I’m on my iPad so sorry about any typos:P

Reply:
I added this https://github.com/luckyframework/lucky/pull/609/commits/159e34b7fe70392449727e4f5b62d19ebf243cdb

As the commit message calls out, it required that I created a new way for the session to bootstrap itself. Lemme know what you think of what I did.

I have a `Lucky::CooikeJar.session_cookie` method which is what you use to initialize a `Lucky::SessionCookie` which seems... wrong? Bad? What do you think? Should I rename the method or class?